clang/include/clang/Sema/Sema.h

```diff
@@ -314,6 +314,7 @@ class SYCLIntegrationHeader {
     kind_accessor = kind_first,
     kind_std_layout,
     kind_sampler,
+    kind_stream,
```

Comment:
Can you append this to list instead? See discussion: https://github.com/intel/llvm/pull/3345#discussion_r592661622